src/assets/locales/fr.json

```diff
@@ -53,30 +74,36 @@
   "button.cancel": "Annuler",
   "button.continue": "Continuer",
   "button.next": "Suivant",
+  "chat-message-input.warning-max-characters": "Caractères max",
   "chat-sidebar-container.input-placeholder.empty-room": "Il n'y a personne pour le moment...",
   "chat-sidebar-container.input-placeholder.empty-room-bot": "Envoyer le message vers {discordChannels}",
   "chat-sidebar-container.input-placeholder.occupants": "{occupantCount, plural, one {Envoyer un message à un autre...} other {Envoyer un message aux {occupantCount} autres...} }",
   "chat-sidebar-container.input-placeholder.occupants-and-bot": "{occupantCount, plural, one {Envoyer un message à un autre et {discordChannels}...} other {Envoyer un message aux {occupantCount} autres et {discordChannels}...} }",
   "chat-sidebar.input.placeholder": "Message...",
-  "chat-sidebar.log-message.audio-normalization-disabled": "l'audioNormalization est désactivée.",
-  "chat-sidebar.log-message.audio-normalization-nan": "la commande audioNormalization nécessite un nombre valide dans les paramètres...",
+  "chat-sidebar.log-message.audio-normalization-disabled": "La normalisation audio est désactivée.",
+  "chat-sidebar.log-message.audio-normalization-nan": "La commande audioNormalization nécessite un nombre valide dans les paramètres...",
+  "chat-sidebar.log-message.audio-resumed": "L'audio a repris.",
```

Comment:
```suggestion
  "chat-sidebar.log-message.audio-resumed": "L'audio a reprit.",
```

Reply:
euh, are you sure? https://leconjugueur.lefigaro.fr/conjugaison/verbe/reprendre.html

Reply:
aha you're right